app/lib/backend/model_extensions/service.rb

```diff
@@ -40,12 +40,6 @@ def delete_backend_service
         System::ErrorReporting.report_error e
         raise e
       end
-
-      private
-
-      def make_default_backend_service
```

Comment:
The task clearly shows that we still need this, so maybe making it public instead?

On the other hand, I guess you decided to remove it to make it clear that making a service the "default" service is not something related to porta per se anymore, right?

Reply:
> I guess you decided to remove it to make it clear that making a service the "default" service is not something related to porta per se anymore

Exactly :smile:

Reply:
I see. I think we should keep it for the moment though. As I said, make it public and add a comment pointing out that we may need this because until we no longer support `provider_key` as an option to authenticate.

Reply:
okidoki :+1:

Reply:
In fact, making a service the default in backend/apisonator should also be reflected in `Account.default_service_id`. I know we are trying to no longer support default services, but apisonator still need them and we may be breaking consistency if we don't follow for now.

Reply:
Ok but we also need to be careful to ONLY do it for those providers that already have a default service and not end up introducing it for new ones... for that I wonder if I should make as you said in https://github.com/3scale/porta/pull/1886#discussion_r428144393 but making the method private to only access it from here 🤔

Reply:
Good point. Let's keep it private then.